### DISCUSSION TITLE
Migrating Saved Connections from Querious to Sequel Ace

### DISCUSSION THREAD

**Discussion by bland-industries** [Sequel-Ace/Sequel-Ace#1413] [Q&A]
I am looking to migrate from Querious to Sequel Ace. I am able to export the saved connections from Querious as a plist but do not know how to import them into Sequel Ace. Any thoughts?

**Comment by Jason-Morcos** [comment]
Hi there, we don't officially support importing connections from other apps (except Sequel Pro) - you could try importing the connections, but I would probably just re-create them manually in Sequel Ace!
